dogfood/coder/main.tf

```diff
@@ -710,4 +744,128 @@ resource "coder_metadata" "container_info" {
+
+    Avoid shortcuts like mocking tests. When you get stuck, you can ask the user
+    but opt for autonomy.
+
+    -- Tool Selection --
+    - coder_report_task: providing status updates or requesting user input.
+    - playwright: previewing your changes after you made them
+      to confirm it worked as expected
+    -	desktop-commander - use only for commands that keep running
+      (servers, dev watchers, GUI apps).
+    -	Built-in tools - use for everything else:
+      (file operations, git commands, builds & installs, one-off shell commands)
+
+    Remember this decision rule:
+    - Stays running? → desktop-commander
+    - Finishes immediately? → built-in tools
+
+    -- Task Reporting --
+    Report all tasks to Coder, following these EXACT guidelines:
+    1. Be granular. If you are investigating with multiple steps, report each step
+    to coder.
+    2. IMMEDIATELY report status after receiving ANY user message
+    3. Use "state": "working" when actively processing WITHOUT needing
+    additional user input
+    4. Use "state": "complete" only when finished with a task
+    5. Use "state": "failure" when you need ANY user input, lack sufficient
+    details, or encounter blockers
+
+    In your summary:
+    - Be specific about what you're doing
+    - Clearly indicate what information you need from the user when in
+    "failure" state
+    - Keep it under 160 characters
+    - Make it actionable
+
+    -- Context --
+    There is an existing application in the current directory.
+    Be sure to read CLAUDE.md before making any changes.
+
+    This is a real-world production application. As such, make sure to think carefully, use TODO lists, and plan carefully before making changes.
+    </system>
+  EOT
+}
+
+resource "coder_env" "claude_task_prompt" {
+  count    = local.has_ai_prompt ? data.coder_workspace.me.start_count : 0
+  agent_id = coder_agent.dev.id
+  name     = "CODER_MCP_CLAUDE_TASK_PROMPT"
+  value    = data.coder_parameter.ai_prompt.value
+}
+
+resource "coder_env" "anthropic_api_key" {
+  count    = local.has_ai_prompt ? data.coder_workspace.me.start_count : 0
+  agent_id = coder_agent.dev.id
+  name     = "ANTHROPIC_API_KEY"
+  value    = var.anthropic_api_key
+}
+
+resource "coder_app" "develop_sh" {
+  count        = local.has_ai_prompt ? data.coder_workspace.me.start_count : 0
+  agent_id     = coder_agent.dev.id
+  slug         = "develop-sh"
+  display_name = "develop.sh"
+  icon         = "${data.coder_workspace.me.access_url}/emojis/1f4bb.png" // 💻
+  command      = "screen -x develop_sh"
+  share        = "authenticated"
+  subdomain    = true
+  open_in      = "tab"
+  order        = 0
+}
+
+resource "coder_script" "develop_sh" {
+  count              = local.has_ai_prompt ? data.coder_workspace.me.start_count : 0
+  display_name       = "develop.sh"
+  agent_id           = coder_agent.dev.id
+  run_on_start       = true
+  start_blocks_login = false
+  script             = <<-EOT
```

Comment:
This script could use an icon too.

Reply:
Addressed in fb0f771c568f56f4488636b39c65416a9c3b0f1c